stdlib/caramel_runtime.ml

```diff
@@ -3,3 +3,5 @@ external recv_with_timeout : int -> 'a = "recv"
 external recv_and_wait : unit -> 'a = "recv"
 
 external ( ^ ) : string -> string -> string = "binary_concat"
+
+external ( |> ) : 'a -> 'b -> 'c = "pipe"
```

Comment:
```suggestion
external ( |> ) : 'a -> ('a -> 'b) -> 'b = "pipe"
```

This should be a stricter signature that matches what we want out of this operator.